models/comments-models.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection")
+
+function getCommentsData(article_id, sort_by = "created_at", order = "desc") {
+    return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY ${sort_by} ${order};`, [article_id]).then((comments) => {
+        if (comments.rows.length === 0) {
+            return Promise.reject({status: 404, message: "Article not found"})
+        }
+        return comments.rows
+    })
```

Comment:
In this model function you're throwing back a `Promise.reject` for any article that has no comments as a `404`
The issue here is that some articles in the database _have no comments attached_ ( see article_id: 2 )
If an article in the database has no comments that should be a `200` response with an empty array `[]`